draft-ietf-quic-transport.md

```diff
@@ -4843,7 +4843,8 @@ Token Length:
 
 Token:
 
-: An opaque blob that the client may use with a future Initial packet.
+: An opaque blob that the client may use with a future Initial packet. The token
+  MUST NOT be empty.
```

Comment:
Did you want to specify the connection close error to use if it's empty?

Reply:
I'd use a FRAME_ENCODING_ERROR error here. However, for other invalid frame errors we use PROTOCOL_VIOLATION. Not sure if that's something we want to address at some point.